### ISSUE TITLE
Placing WWSympa behind reverse proxy

### ISSUE THREAD

**Issue by ikedas** [sympa-community/sympa-community.github.io#35]
"[Placing WWSympa behind reverse proxy](https://git.io/fhZVx)" should be written.

There are at least two ways to accomplish.  See sympa-community/sympa#517.

  - [ ] Establish configuration method with Apache + mod_proxy
  - [ ] Establish configuration method with nginx
  - [ ] Establish configuration method with the other proxies (if possbile)
  - [ ] Write and submit changes on documentation

**Comment by racke** [comment]
I suggest that "Nginx" would be the first method, as it is more efficient and the better choice (IMHO).

**Comment by ikedas** [comment]
I agree to put nginx at the first.  Next, we should provide information for users of popular systems like Apache, and at last, hopefully, for people reluctantly using proprietary systems like Blue Coat.

**Comment by dverdin** [comment]
We have a working configuration for Apache. We use it for Universalistes development.
I can write it easily.

**Comment by ikedas** [comment]
Note that project members (e.g. three of people here are members) can edit files directly from the Code page.  As documentation may be changed more easily than source code, we may work with such "wiki-way" (Of couses you may also descide to submit changes as PR to ask for review by others).

**Comment by dverdin** [comment]
Bam ! done for Apache + mod_proxy.
I used the "wiki" method to edit the files. Way easier like this.
